test.py

```diff
@@ -1,12 +1,38 @@
-from number.FloatNumber import FloatNumber
+# ckrawczyk
+# mrybarczyk
+
 from number.NumberFabric import NumberFabric
 from number.NumberType import NumberType
+from matrix.MyMatrix import MyMatrix
+import numpy
+import random
 
-print("test")
-
+maxRandomValue = 65536
```

Comment:
Widziałem, ze tam dalej dajesz maxRandomValue - 1, to tutaj daj lepiej po prostu 65535

Reply:
W 20. linijce używam całego maxRandomValue tho

Reply:
+ Dokładnie tak stoi w PDF-ie "(...)gdzie r jest pseudolosową liczbą całkowitą z przedziału: {−2^16, 2^16 −1}", więc patrząc zgodnie z poleceniem imo wychodzi czytelniej

Reply:
Nie rozumiem, maxRandomValue to 65535, czyli 2^16 - 1, czyli zamiast robić maxRandomValue - 1, to lepiej wcześniej odjąć te -1, i nie pisać tego w kodzie. Zwłaszcza, gdy zawsze używasz tej zmiennej z -1